src/main/java/org/opentripplanner/middleware/auth/RequestingUser.java

```diff
@@ -67,18 +67,27 @@ static RequestingUser createTestUser(Request req) {
     }
 
     /**
-     * Determine if requesting user is a third party user.
+     * Determine if requesting user is a third party user. A third party user is classed as a user coming via the AWS
+     * API gateway.
      */
-    public boolean isThirdParty() {
+    public boolean isThirdPartyUser() {
+        // TODO: Look to enhance api user check. Perhaps define specific field to indicate this?
         return apiUser != null;
     }
 
     /**
-     * Check if the incoming user is an admin user. To be classed as an admin user, the user must not be any other user
-     * type.
+     * Determine if requesting user is a first party user. A first party user is a user coming directly from MOD UI.
+     */
+    public boolean isFirstPartyUser() {
+        // TODO: Look to enhance otp user check. Perhaps define specific field to indicate this?
+        return otpUser != null;
+    }
```

Comment:
Sorry if my previous comment spurred the creation of this method, but the way I've been thinking about first vs. third party users is in the context of the MOD UI vs. people that sign up for API access. In other words, the distinction between first and third is happening for specific API users (rather than OTP vs. API user). In any case, I think we should just remove this method in favor of null checks for otpUser where they used to be.

Reply:
Reverted.